integration_tests/test_dbapi.py

```diff
@@ -371,3 +371,14 @@ def test_invalid_query_throws_correct_error(trino_connection):
             """,
             params=(3,),
         )
+
+
+def test_eager_loading_cursor_description(trino_connection):
+    cur = trino_connection.cursor()
+    cur.execute('SELECT * FROM system.runtime.nodes')
+    des_before = cur.description
+    assert des_before is not None
+    cur.fetchone()
+    des_after = cur.description
```

Comment:
I would like if the check was a bit more strict and explicitly verified that the columns in the description match the ones requested from the query.

The current check will still pass if we add a bug in the part where cursor.description is set.